sidebar.jsx

```diff
@@ -141,25 +156,28 @@ class Sidebar extends React.Component {
           <button className="sb-smallButton">
             Make an Offer
           </button>
-          <button className="sb-smallButton" id="watchButton" onClick={this.watchProduct.bind(this)}>
+          <button className="sb-smallButton sb-floatRight" id="watchButton" onClick={this.watchProduct.bind(this)}>
             ☆ Watch
           </button>
         </div>
         {this.openToOffer()}
         {this.shippingSpeed()}
-        <div className="sb-grey">--------------------------------------------------</div>
         {this.confidence()}
-        <div className="sb-grey">--------------------------------------------------</div>
-        <div className="sb-smallText sb-bigSpace sb-grey">
-          Shipped From
-          <div className="sb-bold sb-black">{this.state.seller.name}</div>
-          <div>{this.state.seller.address}</div>
-          {this.sellerRaiting(this.state.seller.reviews.rating)}
-          {this.joinedYear()}
-        </div>
+        <section className="sb-smallText">
+          <div className="sb-bigSpace sb-grey sb-half sb-floatLeft">
+            Shipped From
+            <div className="sb-bold sb-black">{this.state.seller.name}</div>
+            <div>{this.state.seller.address}</div>
+            {this.sellerRaiting(this.state.seller.reviews.rating)}
+            {this.joinedYear()}
+          </div>
+          <div className="sb-extraSpace sb-blue">
+            <img src={QuickerShipperImg} alt="" />
+          </div>
+        </section>
         <div>
           <button className="sb-smallButton sb-smallText">Message Seller</button>
-          <button className="sb-smallButton sb-smallText">Payment and Returns</button>
+          <button className="sb-smallButton sb-smallText sb-floatRight">Payment and Returns</button>

```

Comment:
This line exceeds the 80 char limit, same as above. This time you may get away with the following pattern:
This line exceeds 80 characters. Could you use the following pattern to overcome this?
```
<button>
  [content]
</button>
```